code/modules/jobs/job_types/ncr.dm

```diff
@@ -55,18 +69,19 @@ Captain
 
 /datum/outfit/job/ncr/f13captain
 	name = "NCR Captain"
-	jobtype = /datum/job/ncr/f13captain
-	uniform =  		/obj/item/clothing/under/f13/ncr/officer
-	accessory =       /obj/item/clothing/accessory/ncr/CPT
-	suit = 			/obj/item/clothing/suit/armor/f13/ncrarmor/mantle
-	head = 			/obj/item/clothing/head/beret/ncr
-	gloves =		/obj/item/clothing/gloves/combat
-	glasses = 		/obj/item/clothing/glasses/sunglasses/big
-	suit_store = 	/obj/item/gun/ballistic/automatic/pistol/m1911
+	jobtype 	= /datum/job/ncr/f13captain
+	uniform		= /obj/item/clothing/under/f13/ncr/officer
+	accessory 	= /obj/item/clothing/accessory/ncr/CPT
+	suit 		= /obj/item/clothing/suit/armor/f13/ncrarmor/mantle
+	head 		= /obj/item/clothing/head/beret/ncr
+	gloves 		= /obj/item/clothing/gloves/combat
+	glasses 	= /obj/item/clothing/glasses/sunglasses/big
+	suit_store  = /obj/item/gun/ballistic/automatic/marksman
```

Comment:
Do not give the captain the /automatic/marksman. It's a fully-automatic sniper rifle.